command/daemon/daemon.go

```diff
@@ -14,6 +14,7 @@ import (
 	"github.com/drone-runners/drone-runner-kube/engine/resource"
 	"github.com/drone-runners/drone-runner-kube/internal/kube"
 	"github.com/drone-runners/drone-runner-kube/internal/match"
+	"github.com/drone/runner-go/pipeline/uploader"
```

Comment:
Organize imports. Move this line below, where the other runner-go imports are.

Reply:
done